srcs/app/http/HttpResponse.cpp

```diff
@@ -0,0 +1,141 @@
+#include <HttpResponse.hpp>
+#include <sstream>
+#include <stdexcept>
+#include <utility>
+#include <StringUtils.hpp>
+
+const char HttpResponse::kCRLF_[] = "\r\n";
+
+const std::map<std::size_t, std::string>
+HttpResponse::kResponseStatusMap = CreateResponseStatusMap();
+
+const std::map<std::size_t, std::string>
+HttpResponse::CreateResponseStatusMap() {
+	ResponseStatusMap rs;
+
+	rs.insert(std::make_pair(100, "Continue"));
+	rs.insert(std::make_pair(101, "Switching Protocols"));
```

Comment:
Maybe we do not need to implement all codes since we should define a scenario for each of them, but it is great to have most of them as a reference.